src/styles/global/mixins.styl

```diff
@@ -6,10 +6,37 @@
 
 
 // Media Queries
-above(width)
-  @media screen and (min-width: width)
+above($width)
+  @media screen and (min-width: pxToEm($width))
     {block}
 
-below(width)
-  @media screen and (max-width: width)
+below($width)
+
+  @media screen and (max-width: pxToEm($width))
     {block}
+
+
+/*
+ * Transform pixel values to REM/EM
+ */
+
+pxTo($px, $unit)
```

Comment:
Ah the religion of em vs rem vs px, em-based mqs make more sense tho and rem keep it more consistent. This is not really a pain if we use a pxtorem function like the one I have in there. Check this other article out - https://zellwk.com/blog/media-query-units/